src/components/tile-select/tile-select.component.js

```diff
@@ -111,37 +113,41 @@ const TileSelect = ({
           {...rest}
         />
         <StyledTileSelect disabled={disabled} checked={checked}>
-          <StyledTitleContainer>
-            {title && (
-              <StyledTitle {...(typeof title !== "string" && { as: "div" })}>
-                {title}
-              </StyledTitle>
-            )}
+          <Box display="flex" justifyContent="space-between">
+            {prefixAdornment && <Box mr={3}>{prefixAdornment}</Box>}
+            <Box flexGrow="1">
+              <StyledTitleContainer>
+                {title && (
+                  <StyledTitle
+                    {...(typeof title !== "string" && { as: "div" })}
```

Comment:
it would be nice to have a function like
```
(element) => typeof element !== "string" && { as: "div" }
```

Reply:
Yeah that's a good idea I'll update it after another reviewer has looked at the PR